CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 4.0.0 DATE......................................................................
```

Comment:
Shouldn't this have a date?

Reply:
Shit - thanks! I put things in there so I can't miss them when I go to make a pull request. But, I missed it :-(